apps/mobile/src/app/(home)/mpc-wallets.tsx

```diff
@@ -0,0 +1,111 @@
+import { useRef, useState } from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { NotifyUserSheet, OptionData } from '@/components/sheets/notify-user-sheet.layout';
+import { Trans, t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import {
+  Box,
+  Cell,
+  LogoMPCBitgo,
+  LogoMPCCapsule,
+  LogoMPCCopper,
+  LogoMPCFireblocks,
+  LogoMPCFordefi,
+  LogoMPCPortal,
+  LogoMPCPrivy,
+  LogoMPCQredo,
+  QuestionCircleIcon,
+  SheetRef,
+  Text,
+  Theme,
+  TouchableOpacity,
+} from '@leather.io/ui/native';
+
+export default function MPCWalletsScreen() {
+  const { bottom } = useSafeAreaInsets();
+  const theme = useTheme<Theme>();
+  const notifyUserSheetRef = useRef<SheetRef>(null);
+  const [optionData, setOptionData] = useState<OptionData | null>(null);
+  function onOpenNotificationsSheet(option: OptionData) {
+    setOptionData(option);
+    notifyUserSheetRef.current?.present();
+  }
+  function onCloseNotificationsSheet() {
+    setOptionData(null);
+  }
+  function getUnavailableFeatures() {
```

Comment:
Does this have to be a function? It seems like it just returns a config object we could move into another file